### PR TITLE
Update CHANGELOG.md for the 2.3.6 release

### PR DESCRIPTION
**Pull request by daschuer**

### CODE REVIEW

**Comment by daschuer** [comment]
Now the milestone is cleared. Thank you @Swiftb0y for this great final spurt. 

This is a list of PRs targeting the 2.3 branch: 
https://github.com/mixxxdj/mixxx/pulls?q=is%3Aopen+is%3Apr+base%3A2.3+no%3Amilestone
I will re-target them to 2.4 after the release.

File: CHANGELOG.md [hunk 1]
```diff
@@ -56,6 +56,11 @@
 * Don't reject key values from file metadata with non-minor/-major scales.
   [#11001](https://github.com/mixxxdj/mixxx/pull/11001)
   [#10995](https://github.com/mixxxdj/mixxx/issues/10995)
+* Allow playing tracks with durations of more than 6 h
+  [#11511](https://github.com/mixxxdj/mixxx/pull/11511)
+  [#10995](https://github.com/mixxxdj/mixxx/issues/10995)
```
Comment (JoergAtGithub) [inline CHANGELOG.md:61]: Issue number seems to be wrong.

**Comment by JoergAtGithub** [comment]
After merging this, please check that new manual PRs still build. Previous changelog updates with typos in issue reference broke PR builds of the manual.

File: CHANGELOG.md [hunk 2]
```diff
@@ -56,6 +56,11 @@
...
+* Soundtouch latency compensation updated for version 2.1.1 to 2.3
```
Comment (Holzhaus) [inline CHANGELOG.md:62]: Wrong tense. 

```suggestion
* Update latency compensation for Soundtouch version 2.1.1 to 2.3
```

File: CHANGELOG.md [hunk 3]
```diff
@@ -56,6 +56,11 @@
 * Don't reject key values from file metadata with non-minor/-major scales.
   [#11001](https://github.com/mixxxdj/mixxx/pull/11001)
   [#10995](https://github.com/mixxxdj/mixxx/issues/10995)
+* Allow playing tracks with durations of more than 6 h
```
Comment (Holzhaus) [inline CHANGELOG.md:59]:
```suggestion
* Allow playing tracks with durations of more than 6 hours
```

**Review by Swiftb0y — APPROVED** [review]
LGTM.

**Pull request merged**